beacon-chain/node/config_test.go

```diff
@@ -228,3 +231,38 @@ func TestConfigureInterop(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigureArchivalNode(t *testing.T) {
+	params.SetupTestConfigCleanup(t)
+	hook := logTest.NewGlobal()
+
+	app := cli.App{}
+	set := flag.NewFlagSet("test", 0)
+	set.Bool(flags.ArchivalNodeFlag.Name, false, "")
+	set.Int(flags.SlotsPerArchivedPoint.Name, 2048, "")
+	set.Bool(features.SaveFullExecutionPayloads.Name, false, "")
+	set.Bool(backfill.EnableExperimentalBackfill.Name, false, "")
+	set.Uint64(storageFlags.BlobRetentionEpochFlag.Name, 4096, "")
+
+	require.NoError(t, set.Set(flags.ArchivalNodeFlag.Name, "true"))
+	cliCtx := cli.NewContext(&app, set, nil)
+
+	require.NoError(t, configureArchivalNode(cliCtx))
+	assert.LogsContain(t, hook, "Enabling Archival mode on the beacon node")
+	assert.LogsContain(t, hook, "Saving full execution payloads")
+	assert.LogsContain(t, hook, "Enabling backfill on nodes")
```

Comment:
If you commit Radek's comments, these will need to be changed too
```suggestion
	assert.LogsContain(t, hook, "Enabling archival mode on the beacon node")
	assert.LogsContain(t, hook, "Saving full execution payloads")
	assert.LogsContain(t, hook, "Enabling backfill")
```